viewing-data.html

```diff
@@ -481,22 +477,6 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<li><p class="first">Metadata associated with any level from screens to wells to images can contain annotations such as gene or siRNA identifiers.</p>
-
-<p>Annotations can be formatted as hyperlinks to online resources such as <a href="http://www.pombase.org" target="_blank" class="external-url">PomBase</a> or <a href="http://www.ensembl.org/index.html" target="_blank" class="external-url">Ensembl</a>.</p>
-
-<p>Click on the hyperlink to open a new page showing the appropriate online resource entry.</p>
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
-<img alt="../images/viewingDataScreenLink.jpg" class="align-center" src="images/viewingDataScreenLink.jpg" style="width: 100%;" />
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
 <li><p class="first">In OMERO.insight, click on the Screen accordion tab in the left hand panel.</p>
 
 <p>Images in any well can be opened in the full viewer for ROI viewing, addition or analysis.</p>
```

Comment:
Actually the statement goes well with the screenshot below it, guess you can ignore my previous comment!